src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -48,4 +45,16 @@ class PetResource(
         return Response.ok(petService.assignOwnerIdToPet(petId, ownerId)).build()
     }
 
+    @PUT
+    @Path("/adoptOwnerToMultiple/{ownerId}/{companyId}")
+    fun adoptMultiple(@PathParam("ownerId") ownerId: UUID, @PathParam("companyId") companyId: Long, @RequestBody pets: List<UUID>): Response {
+        return Response.ok("The owner have " + petService.adoptMultiple(pets, ownerId, companyId) + " new pets").build()
```

Comment:
It is more readable to first set the `petService.adoptMultiple(pets, ownerId, companyId)` into a val and the use this val in the response